Multisig/UI/Settings/OwnerKeyManagement/GenerateOwnerKey/GenerateKeyFlow.swift

```diff
@@ -55,7 +55,7 @@ class GenerateKeyFlow: AddKeyFlow {
             return false
         }
 
-        return OwnerKeyController.importKey(privateKey, name: name, isDrivedFromSeedPhrase: true)
+        return OwnerKeyController.importKey(privateKey, name: name, type: .deviceGenerated, isDrivedFromSeedPhrase: true)
```

Comment:
Minor typo in `isDrivedFromSeedPhrase`. It should be `isDerivedFromSeedPhrase`.

Reply:
1. typo in isDrivedFromSeedPhrase -> `isDerivedFromSeedPhrase`
2. would you make this flag part of the .importedKey type declaration? Generated keys are always imported from seed phrase. It only applies to imported keys.

Reply:
web3keys are not generated from a seed phrases.

Reply:
web3auth is a different key type

Reply:
I agree, but I won't change that part now and will keep it for future refactor